src/store/SolidStore.ts

```diff
@@ -21,6 +21,13 @@ class SolidStore {
     this.ttlStatus = false;
   }
 
+  handleFiles = async (file: any, session: any): Promise<any> => {
```

Comment:
Same as above, Promise<void> is better in case where you are not returning anything